<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditCatchDialog</class>
 <widget class="QDialog" name="EditCatchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Catch</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="2">
      <widget class="QLabel" name="labelLatitude">
       <property name="text">
        <string>L&amp;atitude:</string>
       </property>
       <property name="buddy">
        <cstring>editLatitude</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QLineEdit" name="editLongitude"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelDescription">
       <property name="text">
        <string>&amp;Description:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="buddy">
        <cstring>editDescription</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLineEdit" name="editWeight"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelPath">
       <property name="text">
        <string>&amp;Path:</string>
       </property>
       <property name="buddy">
        <cstring>editPath</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="labelLength">
       <property name="text">
        <string>&amp;Length (cm):</string>
       </property>
       <property name="buddy">
        <cstring>editLength</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="labelLongitude">
       <property name="text">
        <string>L&amp;ongitude:</string>
       </property>
       <property name="buddy">
        <cstring>editLongitude</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBait"/>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="editLatitude"/>
     </item>
     <item row="4" column="1" colspan="4">
      <widget class="QLineEdit" name="editPath"/>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="labelWeight">
       <property name="text">
        <string>&amp;Weight (g):</string>
       </property>
       <property name="buddy">
        <cstring>editWeight</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelCatchTime">
       <property name="text">
        <string>Catch &amp;time:</string>
       </property>
       <property name="buddy">
        <cstring>editCatchTime</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateTimeEdit" name="editCatchTime"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelBait">
       <property name="text">
        <string>&amp;Bait:</string>
       </property>
       <property name="buddy">
        <cstring>comboBait</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QPushButton" name="buttonSelect">
       <property name="text">
        <string>&amp;Select</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="editLength"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelSpecies">
       <property name="text">
        <string>Spe&amp;cies:</string>
       </property>
       <property name="buddy">
        <cstring>comboSpecies</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboSpecies"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="labelSession">
       <property name="text">
        <string>Session:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="comboSession">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="5">
      <widget class="QPlainTextEdit" name="editDescription"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dialogButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editCatchTime</tabstop>
  <tabstop>comboSession</tabstop>
  <tabstop>comboSpecies</tabstop>
  <tabstop>editLength</tabstop>
  <tabstop>editWeight</tabstop>
  <tabstop>comboBait</tabstop>
  <tabstop>editLatitude</tabstop>
  <tabstop>editLongitude</tabstop>
  <tabstop>editPath</tabstop>
  <tabstop>buttonSelect</tabstop>
  <tabstop>editDescription</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>dialogButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditCatchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
